packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -418,26 +422,30 @@ class _DragAvatar<T> extends Drag {
     _entry.markNeedsBuild();
     HitTestResult result = new HitTestResult();
     WidgetFlutterBinding.instance.hitTest(result, globalPosition + feedbackOffset);
-    _DragTargetState<T> target = _getDragTarget(result.path);
-    if (target == _activeTarget)
+    List<_DragTargetState<T>> targets = _getDragTargets(result.path);
+    if (targets.contains(_activeTarget))
       return;
     if (_activeTarget != null)
       _activeTarget.didLeave(data);
-    _activeTarget = target;
+    _activeTarget = targets.firstWhere(
+      (_DragTargetState<T> target) => target.willAccept(data),
+      orElse: () => targets.isEmpty ? null : targets.first
+    );
     _activeTargetWillAcceptDrop = _activeTarget != null && _activeTarget.didEnter(data);
```

Comment:
this calls onWillAccept twice on every successful node. that seems bad.

Maybe just call didEnter on every node during the search?